serving/installing-logging-metrics-traces.md

```diff
@@ -1,73 +1,65 @@
 # Monitoring, Logging and Tracing Installation
 
-Knative Serving offers two different monitoring setups: 
-One that uses Elasticsearch, Kibana, Prometheus and Grafana and 
-another that uses Stackdriver, Prometheus and Grafana. See below 
-for installation instructions for these two setups. You can install 
-only one of these two setups and side-by-side installation of these two are not supported.
+Knative Serving offers two different monitoring setups:
+[Elasticsearch, Kibana, Prometheus and Grafana](#Elasticsearch,-Kibana,-Prometheus-&-Grafana-Setup) or [Stackdriver, Prometheus and Grafana](#Stackdriver,-Prometheus-&-Grafana-Setup). You can install only one of these two setups and side-by-side installation of these two are not supported.
 
 ## Elasticsearch, Kibana, Prometheus & Grafana Setup
 
-*If you installed Knative Serving using [Easy Install](../install/README.md#Installing-Knative) guide, 
-skip this step and continue to [Create Elasticsearch Indices](#Create-Elasticsearch-Indices)*
+If you installed the [latest Knative Serving components](../install/README.md#Installing-Knative),
+skip this step and continue to [Create Elasticsearch Indices](#Create-Elasticsearch-Indices)
 
-
-Run:
-
-```shell
+* Install Knative monitoring components:
+```
 kubectl apply -R -f config/monitoring/100-common \
     -f config/monitoring/150-elasticsearch \
     -f third_party/config/monitoring/common \
     -f third_party/config/monitoring/elasticsearch \
     -f config/monitoring/200-common \
     -f config/monitoring/200-common/100-istio.yaml
 ```
-
-Monitor logging & monitoring components, until all of the components report Running or Completed:
-
-```shell
+* The installation is complete when logging & monitoring components are all reported `Running` or `Completed`:
+```
 kubectl get pods -n monitoring --watch
 ```
-
+```
+NAME                                  READY     STATUS    RESTARTS   AGE
+elasticsearch-logging-0               1/1       Running   0          2d
+elasticsearch-logging-1               1/1       Running   0          2d
+fluentd-ds-5kc85                      1/1       Running   0          2d
+fluentd-ds-vhrcq                      1/1       Running   0          2d
+fluentd-ds-xghk9                      1/1       Running   0          2d
+grafana-798cf569ff-v4q74              1/1       Running   0          2d
+kibana-logging-7d474fbb45-6qb8x       1/1       Running   0          2d
+kube-state-metrics-75bd4f5b8b-8t2h2   4/4       Running   0          2d
+node-exporter-cr6bh                   2/2       Running   0          2d
+node-exporter-mf6k7                   2/2       Running   0          2d
+node-exporter-rhzr7                   2/2       Running   0          2d
+prometheus-system-0                   1/1       Running   0          2d
+prometheus-system-1                   1/1       Running   0          2d
+```
 CTRL+C when it's done.
 
-### Create Elasticsearch Indices
```

Comment:
Any reason to remove this part? These steps are required to complete the installation.

Reply:
I thought it would be best to have all the Kibana information in the accessing-logs file; however I can see that the elasticsearch index creation is better in this file because it has to do with setup and now just viewing.